<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color:rgb(255, 253, 229);
}
QFrame{
	border:none;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="10">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="panelInferior">
      <property name="styleSheet">
       <string notr="true">#Carpeta, #Permisos {
border: 1px solid rgb(235, 201, 118);
}
QPushButton{
	background-color:rgb(35, 255, 200);
}
QPushButton:hover{
	background-color:rgb(80, 214, 174);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="panelRuta">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFrame" name="paneSuperior">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btnRegresar">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgb(191, 42, 53)
}
QPushButton:hover{
	background-color:rgb(211, 42, 53)
}</string>
               </property>
               <property name="text">
                <string>Regresar</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../src/img/volver-icon.png</normaloff>../src/img/volver-icon.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtRuta">
               <property name="styleSheet">
                <string notr="true">background-color:white;
color:gray;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>/</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="PanelContenidoInferior">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="Carpeta">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="lblTituloCarpeta">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Carpeta</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtNombreCarpeta">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:white;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblNombreCarpeta">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Nombre:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeWidget" name="arbolPrincipal">
               <property name="styleSheet">
                <string notr="true">background-color:white;</string>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContents</enum>
               </property>
               <property name="autoScroll">
                <bool>true</bool>
               </property>
               <property name="indentation">
                <number>20</number>
               </property>
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="headerVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="headerDefaultSectionSize">
                <number>206</number>
               </attribute>
               <column>
                <property name="text">
                 <string>Nombre</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Tipo</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Ruta</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnCrearCarpeta">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Crear</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="Permisos">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="lblTituloPermisos">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Permisos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbNombreUsuario">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="toolTipDuration">
                <number>-1</number>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:rgb(240, 240, 240);</string>
               </property>
               <property name="editable">
                <bool>false</bool>
               </property>
               <property name="currentText">
                <string/>
               </property>
               <property name="currentIndex">
                <number>-1</number>
               </property>
               <property name="maxVisibleItems">
                <number>10</number>
               </property>
               <property name="maxCount">
                <number>10</number>
               </property>
               <property name="insertPolicy">
                <enum>QComboBox::NoInsert</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
               <property name="duplicatesEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblNombreUsuario">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Nombre usuario: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnAgregarUsuario">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Agregar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbtnEscritura">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Escritura</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbtnLectura">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Lectura</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbtnEjecucion">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Ejecucion</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblTituloListView">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Tabla de usuarios</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="listPermisosUsuarios">
               <property name="styleSheet">
                <string notr="true">background-color:white;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnEliminarUsuario">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Eliminar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
